src/customer/customer.d.ts

```diff
@@ -19,7 +65,7 @@ export = class Customer {
     referenceID: string;
     mobileNumber?: string;
     email?: string;
-    givenNames: string;
+    givenNames?: string;
```

Comment:
shouldn't this field be mandatory?

Reply:
for v20200519, yes it's mandatory
but for v20201031, we don't have that field anymore, it will be under individualDetail